bean-utils-library/src/test/java/com/hotels/beans/transformer/ImmutableObjectTransformationTest.java

```diff
@@ -485,6 +488,26 @@ public void testFieldTransformationSkipWorksProperly() {
         underTest.resetFieldsTransformationSkip();
     }
 
+    /**
+     * Test that the transformer function is applied earlier than the default value
+     */
+    @Test
+    public void testTransformerFunctionHasHigherPriorityThanDefaultValue() {
+        //GIVEN
+        BeanUtils beanUtils = new BeanUtils();
+        FromFooSimpleBooleanField fromFooSimpleNullFields = new FromFooSimpleBooleanField();
+        FieldTransformer<Boolean, Boolean> nullToTrue =
+            new FieldTransformer<>("work", aBoolean -> aBoolean == null || aBoolean);
+
+        //WHEN
+        ImmutableToFooSimpleBoolean transformed = beanUtils.getTransformer()
+            .withFieldTransformer(nullToTrue)
+            .transform(fromFooSimpleNullFields, ImmutableToFooSimpleBoolean.class);
+
+        //THEN
+        assertTrue(transformed.getWork());
```

Comment:
please add the following instruction after the `assertTrue`:
```
underTest.resetFieldsTransformer();
```
this will remove the transformer function defined in this method in order to let the other tests not influenced by this